GSA/usagov-benefits-eligibility
# 767 top

## PR Summary

Add "Return to top" link in footer.

## Related Github Issue

- fixes #767

## Type of change

Please delete options that are not relevant.


## Branch Name

767-top

## Testing environment

https://federalist-edd11e6f-8be2-4dc2-a85e-1782e0bcb08e.sites.pages.cloud.gov/preview/gsa/usagov-benefits-eligibility/767-top/death-of-a-loved-one/?

## Detailed Testing steps

Link to testing steps in the issue or list them here:

1. Navigate to https://federalist-edd11e6f-8be2-4dc2-a85e-1782e0bcb08e.sites.pages.cloud.gov/preview/gsa/usagov-benefits-eligibility/767-top/death-of-a-loved-one/?
2. Click "Top" in bottom right of footer.
3. Check it to return to top.

etc...

## Checklist:
